tests/src/core/testqgis.cpp

```diff
@@ -359,6 +360,36 @@ void TestQgis::testQgsRound()
   QGSCOMPARENEAR( qgsRound( 9999999.87654321987654321, 14 ), 9999999.876543219876543, 0.00000000000001 );
 }
 
+void TestQgis::testQgsVariantEqual()
+{
+
+  // Invalid
+  QVERIFY( qgsVariantEqual( QVariant(), QVariant() ) );
+  QVERIFY( QVariant() == QVariant() );
```

Comment:
Just wondering where you want to go with those checks... Trigger a warning when Qt upstream introduces changes to `QVariant::operator==`?

Reply:
no, it's just to confirm that I'm only changing the comparison with NULL vs 0